prelude/prelude.purs

```diff
@@ -34,10 +34,14 @@ module Prelude
   otherwise = true
 
   -- | Flips the order of the arguments to a function of two arguments. 
+  -- |
+  -- | `flip (/) 6 3 == 3 / 6`
```

Comment:
What I found when typing it up is that using the four-spaces method made it really easy to accidentally combine the examples with the function type signature in the rendered MD.  The backticks were a way to handle that in a visually unambiguous way.  I prefer @garyb suggestion as a good alternative to try.